### ISSUE TITLE
Shuttles need a blank Z level to jump into

### ISSUE THREAD

**Issue by CakeQ** [Apollo-Community/ApolloStation#1013]
For if they run out of fuel on the overmap, or if the pilot goes offline.

**Comment by DreadedReborn** [comment]
Or they could be given the power of space to just drift naturally. I remember one codebase having spacepods just float in a direction unless you thrust in the opposite direction.

**Comment by astralenigma** [comment]
Yeah, but he's talking about shuttles stopping in the overmap, honestly what he's saying is that there's no way to save the pod if it loses power while it isn't in neither of the playable maps and let the pods keep on drifting only makes them harder to recover.
Creating a map for the lost pod though seems kinda of unreasonable performance wise as you either have a map for each sector or you can abuse it by getting 2 pods going to the same Z-Level from different sectors.
What needs to be done in my opinion is add some kind of tow-pod to bring back any ship that isn't tagged to the teleporters and add other overmap interactions.

**Comment by CakeQ** [comment]
A simple suggestion would be to add an emergency teleport that would teleport the pod to a random z-level (in a list) into anywhere that has area/space which uses up all fuel+power
